src/classes/TestResult.js

```diff
@@ -186,7 +206,7 @@ export default class TestResult extends BubblingEventTarget {
 		if (test.throws !== undefined) {
 			Object.assign(this, this.evaluateThrown());
 		}
-		else if (test.maxTime || test.maxTimeAsync) {
+		else if (!("expect" in test) && (test.maxTime || test.maxTimeAsync)) {
```

Comment:
Why are we only evaluating time taken when the test has no `throws` and no `expect`? It seems to me that if the test took too long, we'd want to know *anyway*? Especially if the reason it failed is that it timed out?

It _seems_ to me (at first glance) that is `if ... else if ... else` needs to become separate conditions really.

Reply:
I agree. I refactored the code so that we evaluated the time taken by a test, regardless of its type.